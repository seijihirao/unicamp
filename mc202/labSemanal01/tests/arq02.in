5
9.237101 
-0.080893 3.933732 
9.086825 -2.378481 4.947153 
5.493106 -5.995869 4.425407 -7.546102 
4.184390 9.494412 6.127104 -4.025851 0.557015 
