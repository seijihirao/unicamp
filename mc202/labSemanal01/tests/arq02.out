1.267198965791 
-0.484736986795 0.270078467438 
1.238944608983 -0.916721276700 0.460618264417 
0.563266397601 -1.596849648928 0.362521444917 -1.888318920918 
0.317206289371 1.315577661023 0.682468437232 -1.226453900504 -0.364799802928 

2.497269933333 5.318684159799 
